locale/en/LC_MESSAGES/django.po

```diff
@@ -2441,7 +2443,9 @@ msgstr "Batch create and assess submissions"
 
 #: edit_course/templates/edit_course/batch_assess.html
 #, python-format
-msgid "BATCH_CREATE_AND_ASSESS_JSON_FORM_INSTRUCTIONS -- %(id)s, %(exercises_api_url)s"
+msgid ""
+"BATCH_CREATE_AND_ASSESS_JSON_FORM_INSTRUCTIONS -- %(id)s, "
+"%(exercises_api_url)s"
```

Comment:
How did these changes appear in this pull request?

Reply:
I think the line lengths weren't short enough when those changes were made, so running `makemessages` shortened them automatically now.